test/layer_normalization_test.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+import {layerNormalization} from '../src/layer_normalization.js';
+import {Tensor} from '../src/lib/tensor.js';
+import * as utils from './utils.js';
+
+
+describe('test layerNormalization', function() {
+  function testLayerNorm(
+      input, expected, scale = undefined, bias = undefined, axes = undefined, options = {}) {
+    const inputTensor = new Tensor(input.shape, input.value);
+    if (scale) {
+      options.scale = new Tensor(scale.shape, scale.value);
+    }
+    if (bias) {
+      options.bias = new Tensor(bias.shape, bias.value);
+    }
+    if (axes) {
+      options.axes = axes;
+    }
+    const outputTensor = layerNormalization(inputTensor, options);
+    console.log('output:', outputTensor);
+    utils.checkShape(outputTensor, input.shape);
+    utils.checkValue(outputTensor, expected);
+  }
+
+  it('layerNormalization default 2D', function() {
+    testLayerNorm(
+        {
+          shape: [2, 3],
+          value: [1, 2, 3, 3, 6, 24],
+        },
+        [-1.2247356859083902,
+          0,
+          1.2247356859083902,
+          -0.8626621354727335,
+          -0.5391638346704585,
+          1.4018259701431919,
+        ],
+    );
+  });
+
+  it('layerNormalization default 3D', function() {
+    testLayerNorm(
+        {
+          shape: [2, 2, 3],
+          value: [1, 2, 3, 6, 5, 4, 3, 6, 24, -10, 0, 5],
+        },
+        [-1.4638475999719223, -0.8783085599831534, -0.29276951999438444,
+          1.4638475999719223, 0.8783085599831534, 0.29276951999438444,
+          -0.1645769966453613, 0.131661597316289, 1.9090931610861905,
+          -1.4482775704791793, -0.46081559060701155, 0.032915399329072226,
+        ],
+    );
+  });
+
+  it('layerNormalization default 3D with axes=[2]', function() {
+    testLayerNorm(
+        {
+          shape: [2, 2, 3],
+          value: [1, 2, 3, 6, 5, 4, 3, 6, 24, -10, 0, 5],
+        },
+        [-1.2247356859083902, 0, 1.2247356859083902,
+          1.2247356859083902, 0, -1.2247356859083902,
+          -0.8626621354727335, -0.5391638346704585, 1.4018259701431919,
+          -1.3363060377513567, 0.26726120755027133, 1.0690448302010853,
+        ],
+        undefined,
+        undefined,
+        [2],
+    );
+  });
+
+  it('layerNormalization 3D with scale and axes=[2]', function() {
+    testLayerNorm(
+        {
+          shape: [2, 2, 3],
+          value: [1, 2, 3, 6, 5, 4, 3, 6, 24, -10, 0, 5],
+        },
+        [-2.4494713718167804, 0, 4.898942743633561,
```

Comment:
```suggestion
        [
          -2.4494713718167804, 0, 4.898942743633561,
```

New line for consistency with elsewhere, like below:

        [
          -1.4494713718167804,
          2,
          7.898942743633561,
          3.4494713718167804,

Reply:
ok